src/txadmission.cpp

```diff
@@ -52,6 +52,16 @@ CTransactionRef CommitQGet(uint256 hash)
     return it->second.entry.GetSharedTx();
 }
 
+static inline uint256 IncomingConflictHash(const COutPoint &prevout)
+{
+    uint256 hash = prevout.hash;
+    unsigned char *first = hash.begin();
+    *first ^= (unsigned char)(prevout.n & 255);
+    first += 8;
+    *first ^= (unsigned char)(prevout.n & 255);
+    return hash;
```

Comment:
all the idx you show are different but your comments are saying that they are the same?

Reply:
all of the idx's are for just one fast filter entry...I was just giving you the results for the hash entries above. So in other words, each of the xor'd hashes gives you all the same 16 idx/idx2 results even though the hashes are all different.

Reply:
ok I think I know the problem.  Although I separated this PR from the fastfilter enhancements PR #1353, you are right that it is very likely that old fastfilter (what is currently on dev) won't use these bits because it just uses a random (but not the first) 32 bits from the hash.

Reply:
ok, i probably should have pulled in ff2 and tested with that...i'll do that now.